src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -148,6 +150,17 @@ export class ArtworkSidebarCommercialContainer extends React.Component<
       this.props.mediator.trigger("launchInquiryFlow", {
         artworkId: this.props.artwork.id,
       })
+
+    const { tracking } = this.props
+    const trackingData = {
+      context_module: "Sidebar",
+      subject: "Contact Gallery",
+      type: "Button",
+      artwork_id: this.props.artwork._id,
+      action: "Click",
+    }
```

Comment:
@mzikherman - do you plan on doing a pass using our new format? All of these string values need to go in `Artsy/Analytics/Schema` (some values are already there) and the tracking should to be attached via a decorator.

Reply:
I thought this was the new way? (vs mediator triggered in Force).

I can move the string values to the schema file (I'm realizing as I'm going along they belong there), but the decorator stuff is really confusing to me. Is there something objectionable to this more imperative version? It seems a lot clearer.

Reply:
I agree that this seems a little simpler in practice, but we've been moving tracking to decorators over the past year, most other code (except publishing) follows that format, and the same model is in place in Emission. I already fought this battle and lost if you feel like digging through the issues.